framework/libra-framework/sources/ol_sources/ol_account.move

```diff
@@ -363,13 +378,20 @@ module ol_framework::ol_account {
         let decimal_places = coin::decimals<LibraCoin>();
         let scaling = math64::pow(10, (decimal_places as u64));
         let value = fixed_point32::create_from_rational(unscaled_value, scaling);
-        // multply will TRUNCATE.
+        // multiply will TRUNCATE.
         let integer_part = fixed_point32::multiply_u64(1, value);
 
         let decimal_part = unscaled_value - (integer_part * scaling);
 
         (integer_part, decimal_part)
     }
+
+    fun scale_from_human(human: u64): u64 {
```

Comment:
Checking this function is included intentionally because afaics it isn't called.